Automattic/jetpack
# Add/connection verified errors react initial state

#### Changes proposed in this Pull Request:
Add the connection verified errors to the connection React initial state. This is part of a bigger project (1202697549597858-as) to provide consistent failed connection error reporting and an interface to reconnect (1202697549597858-as-1202697549597864). 

The changes will affect the following:
* Jetpack
* Connection

#### Other information:


#### Jetpack product discussion
* p9dueE-5w8-p2

#### Does this pull request change what data or activity we track or use?
* No

#### Testing instructions:
* Go to Jetpack > Dashboard.
* Open up the browser's developer console.
* Input `JP_CONNECTION_INITIAL_STATE` and press enter.
* Verify that the JS object that is output has a member named `connectionVerifiedErrors`, and that it's empty.
* Go to "Jetpack Debug -> XML-RPC Errors" and create a verified error.
* Go back to the Jetpack Dashboard and confirm that `JP_CONNECTION_INITIAL_STATE.connectionVerifiedErrors` now contains the newly created error.
